onchain/.changeset/mean-news-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"@cartesi/rollups": major
+---
+
+Inputs are now blockchain-agnostic and self-contained blobs. For example, inputs added by EVM contracts like `InputBox` are encoded using `abi.encodeWithSignature` and contain EVM-specific metadata like `msg.sender` and `block.timestamp`.
```

Comment:
The fact that inputs are encoded with `abi.encodeWithSignature` is a mere implementation detail. Also, we might use `abi.encodeCall` instead for type safety reasons. Either way, let's focus on the fact that inputs now contain metadata.

```suggestion
Inputs are now blockchain-agnostic and self-contained blobs. For example, inputs added by EVM contracts like `InputBox` contain EVM-specific metadata like `msg.sender` and `block.timestamp`.
```